lib/tasks/karma.rake

```diff
@@ -1,14 +1,22 @@
 namespace :karma  do
-  task :start => :environment do
+  task :start => :environment do |task|
+    continue_only_in_test_env task
     with_tmp_config :start
   end
 
-  task :run => :environment do
+  task :run => :environment do |task|
+    continue_only_in_test_env task
     with_tmp_config :start, "--single-run"
   end
 
   private
 
+  def continue_only_in_test_env task
+    if Rails.env != 'test'
+      raise "Task must be called in test environment:\n  bundle exec rake #{task.name} RAILS_ENV=test"
```

Comment:
Here is what it looks like. It should only happen when devs execute directly the rake task. I'm open to suggestions on this.
```
$ bundle exec rake karma:run
rake aborted!
Task must be called in test environment:
  bundle exec rake karma:run RAILS_ENV=test
/home/emak/code/openfoodnetwork/lib/tasks/karma.rake:16:in `continue_only_in_test_env'
/home/emak/code/openfoodnetwork/lib/tasks/karma.rake:8:in `block (2 levels) in <top (required)>'
/home/emak/.rbenv/versions/2.1.5/bin/ruby_executable_hooks:15:in `eval'
/home/emak/.rbenv/versions/2.1.5/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => karma:run
```